src/app/services/modificadores/modificadores.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Server } from '@andes/shared';
+
+
+@Injectable()
+export class ModificadoresService {
+
+    private apiUrl = '/modules/modificadores';  // URL to web api
+
+    constructor(private server: Server) { }
+
+    search(params: any): Observable<any> {
+        return this.server.post(this.apiUrl, params);
```

Comment:
Estoy de acuerdo con @kroce, en realidad no estás creando nada. Probaste usar algo como: return this.server.get(this.apiUrl, params) 
Ya que POST es para create y sino confunde. En la api no usa resourceBase por algún motivo?